docs/source/usage.rst

```diff
@@ -0,0 +1,351 @@
+Usage
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This page shows the most frequent use-cases when using the library. The models available allow for many different
+configurations and a great versatility in use-cases. The most simple ones are presented here, showcasing usage
+for tasks such as question answering, sequence classification, named entity recognition and others.
+
+These examples leverage auto-models, but feel free to modify the code to be more specific and adapt it to your specific
+use-case.
```

Comment:
What are Auto-Models? (A way to instantiate the correct architecture of model automatically)